docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+services:
+  monolith_database:
+    container_name: monolith_db
+    image: mysql/mysql-server:5.7
+    restart: on-failure
+    ports:
+    - "3307:3306"
+    environment:
+      MYSQL_ROOT_PASSWORD:
```

Comment:
@mcruzdev usually it is fine not to include this here in github.. but if we are using the default passwords I would just add them here.. with some comment saying that those are the default ones